miso-web/src/main/webapp/scripts/list.js

```diff
@@ -256,7 +256,7 @@ ListUtils = (function() {
           }) ? " checked=\"checked\"" : "";
 
           return "<input type=\"checkbox\" id=\"" + elementId + "_toggle" + data
-              + "\" onclick=\"ListUtils._checkEventHandler(this.checked, event, " + data + ", '" + elementId + "')\"" + checked + ">";
+              + "\" onclick='ListUtils._checkEventHandler(this.checked, event, " + JSON.stringify(data) + ", \"" + elementId + "\")'" + checked + ">";
```

Comment:
MISO normally does single quotes for JS; did your Eclipse formatter replace the single quotes with double quotes around `elementId`?